src/components/panels/PanelsWrapper.vue

```diff
@@ -1,29 +1,71 @@
 <template>
-  <div class="panels-wrapper t-flex t-flex-col md:t-overflow-auto t-flex-1 md:t-flex-row md:t-space-x-4 t-space-y-4 md:t-space-y-0 t-px-4 lg:t-px-6 md:t-pb-4">
-    <Panel
-      v-for="(panel, i) in panels"
-      v-show="panel.show"
-      :key="`pc${i}`"
-      :panel="panel"
-      :active-view="getActiveView(i)"
-      class="t-flex-1 t-max-h-[500px] md:t-max-h-screen md:t-h-auto"
-      @active-view="onActiveViewChange($event, i)"
-    />
+  <div
+    class="panels-wrapper t-flex-1 t-space-y-4
+    md:t-space-y-0 t-px-4 lg:t-px-6 t-pb-2 lg:t-pb-4 t-overflow-y-hidden lg:t-overflow-hidden"
+  >
+    <div
+      ref="container"
+      class="t-flex t-h-full t-overflow-x-auto t-pb-2 lg:t-pb-0 -t-mr-4 lg:t-mr-0"
+    >
+      <Panel
+        v-for="(panel, i) in panels"
+        v-show="panel.show"
+        :key="`pc${i}`"
+        :panel="panel"
+        :active-view="getActiveView(i)"
+        :default-width="defaultWidth"
+        :class="{ 't-ml-4': i > 0 && !isMobile, 't-ml-2': i > 0 && isMobile, 't-mr-4': i === panels.length - 1 && isMobile }"
```

Comment:
a question about the syntax of conditional styling: do we apply a certain style when the part after `:` is true ?

Reply:
yes, exactly